qiskit/transpiler/passes/mapping/enlarge_with_ancilla.py

```diff
@@ -43,16 +43,18 @@ def run(self, dag):
         Raises:
             TranspilerError: If there is not layout in the property set or not set at init time.
         """
-        if self.layout is None:
-            if self.property_set["layout"]:
-                self.layout = self.property_set["layout"]
-            else:
-                raise TranspilerError(
-                    "EnlargeWithAncilla requires self.property_set[\"layout\"] to run")
+        if self.property_set["layout"] is not None:
+            layout = self.property_set["layout"]
+
+        if self.layout is not None:
+            layout = self.layout
```

Comment:
can just use `self.layout` in these lines instead of defining a `layout`. this should be equivalent to the above 4 lines:
```
self.layout = self.layout or self.property_set.get('layout')
```

Reply:
corrected in 71ea8e71